### PR TITLE
openssl@1.1: Bump cacert.pem to 2019-10-16

### PR DESCRIPTION
**Pull request by xama5**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/linuxbrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/linuxbrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

### CODE REVIEW

**Comment by dawidd6** [comment]
Bump revision and delete bottle line please.

**Comment by xama5** [comment]
@dawidd6 Done. Didn't think it was necessary as the .pem file is added post-installation and not during build.

**Comment by dawidd6** [comment]
Yea, but if we want users to get those updated certs soon, we need to bump revision.

File: Formula/openssl@1.1.rb [hunk 1]
```diff
@@ -6,13 +6,13 @@ class OpensslAT11 < Formula
   mirror "https://www.mirrorservice.org/sites/ftp.openssl.org/source/openssl-1.1.1d.tar.gz"
   sha256 "1e3a91bc1f9dfce01af26026f856e064eab4c8ee0a8f457b5ae30b40b8b711f2"
   version_scheme 1
+  revision 1
```
Comment (iMichka) [inline Formula/openssl@1.1.rb]: `revision 1 unless OS.mac?`

File: Formula/openssl@1.1.rb [hunk 2]
```diff
@@ -25,9 +25,9 @@ class OpensslAT11 < Formula
   unless OS.mac?
     resource "cacert" do
       # homepage "http://curl.haxx.se/docs/caextract.html"
-      url "https://curl.haxx.se/ca/cacert-2017-01-18.pem"
-      mirror "http://cdn.rawgit.com/sjackman/e4066d2cb6b45fbb6d213e676cb109d0/raw/58964378cb5eefe96cba245ef863c57fb2b480e0/cacert-2017-01-18.pem"
-      sha256 "e62a07e61e5870effa81b430e1900778943c228bd7da1259dd6a955ee2262b47"
+      url "https://curl.haxx.se/ca/cacert-2019-10-16.pem"
+      mirror "https://gist.githubusercontent.com/xama5/3321c7e2976872d1bd1e90a41b75fc68/raw/8568685680f1d31ec6613edecd2a83452d84b09b/cacert-2019-10-16.pem"
```
Comment (iMichka) [inline Formula/openssl@1.1.rb]: Can you use my gist instead:
https://gist.githubusercontent.com/iMichka/5a40e35a4d83892b4d5cdd917e7cbd25/raw/1390d7d93ca770366e66e61dd39abca09123eb4e/cacert-2019-10-16.pem

Ideally we would host it somewhere else, but I prefer having the file linked to a maintainer account.

**Comment by dawidd6** [comment]
Thanks for your contribution!

**Pull request closed without merging**